### ISSUE TITLE
Understanding Question from the Paper

### ISSUE THREAD

**Issue by XMaster96** [IDEA-Research/DWPose#10]
He, I read the paper and I kinda have now some open questions.

1. For the first distillation did you pre-train the teacher model or did you took some off the shelf pre-trained model for that.

2. Did you use at any stage the actual real labels from COCO or did you purley trains with labels provided by other models?

3. What Model architecture did you use for your Backbone? How many params?

Thanks for the help in advance.

**Comment by yzd-v** [comment]
1. We train a larger teacher model on COCO and UBody first. And then we use the teacher model to guide a smaller student.
2. For the first stage distillation, we use the GT label from COCO and teacher's logits together. For the second stage distillation, we only use teacher's logits.
3. The backbone is CSPNeXt. You can learn about it in [RTMDet](https://arxiv.org/pdf/2212.07784.pdf).
